tensorflow/lite/kernels/slice_test.cc

```diff
@@ -309,6 +309,74 @@ TEST_P(SliceOpTest, SliceBool) {
   EXPECT_THAT(m.GetOutput(), ElementsAreArray({false, true}));
 }
 
+TEST_P(SliceOpTest, SliceFloat16) {
+  SliceOpModel<Eigen::half, int32_t> m({3, 2, 3, 1}, {4}, {1, 0, 0, 0}, {4},
```

Comment:
Please add include for Eigen/core

Reply:
Addressed the requested change in [#55aee76](https://github.com/MCW-Dev/tensorflow/pull/33/commits/55aee761ece414a6958216f60479cd0358f1b3f3)